test/test_output_engine.py

```diff
@@ -470,6 +471,11 @@ def test_output_csv(self):
         expected_value = [dict(x) for x in reader]
         self.assertEqual(expected_value, self.FORMATTED_OUTPUT)
 
+    @unittest.skipUnless(
+        importlib.util.find_spec("reportlab") is not None
+        and importlib.util.find_spec("pdftotext") is not None,
+        "Skipping PDF tests",
```

Comment:
```suggestion
        "Skipping PDF tests. Please install reportlab and pdftotext to run these tests.",
```

Reply:
Slightly longer message that hopefully explains why the tests are being skipped.